src/components/outline-nav/components/outline-nav-from-headings/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { OutlineHeading, OutlineLinkItem } from 'components/outline-nav/types'
+import { OutlineNavWithActive } from 'components/outline-nav/components'
+import { useMemo } from 'react'
+
+/**
+ * Given a flat array of headings,
+ * Return an array of outline items, for use with OutlineNav components.
+ */
+function buildOutlineItems(headings: OutlineHeading[]): OutlineLinkItem[] {
+	return headings.map((heading: OutlineHeading) => {
+		/* TODO: maybe consider resolving the backticks issue closer to server? */
+		const titleWithoutBackticks = heading.title.replace(/`/g, '')
```

Comment:
👍🏻  Nodding to this TODO comment. Ideally this could happen in the heading generation function? Perhaps this could be a note to pick up once you actually integrate it into the sidebar in the next PR.

Reply:
Great call - I think this'll be a relatively high-effort refactor, since it'll involve moving the abstraction from client-side to server-side, which will involve changing a few more places. I've added a follow-up task 🎟 [Clean up headings-related OutlineNav uses, to move formatting closer to server](https://app.asana.com/0/0/1204024966433303/f) to do this at some point (hopefully soon!)